Library/Homebrew/cli_parser.rb

```diff
@@ -4,10 +4,17 @@
 module Homebrew
   module CLI
     class Parser
+      # rubocop:disable Style/ClassVars
+      @@args = OpenStruct.new
```

Comment:
@MikeMcQuaid Storing this in `Homebrew::CLI::Parser` class as class variable currently, as Parser is not going to be inherited but used as is. 
If we'd consider class variables evil, I'll move this to `Homebrew` class instance variable as discussed on slack.

Reply:
> If we'd consider class variables evil

Yup 👍 

> I'll move this to Homebrew class instance variable as discussed on slack.

To clarify to other readers here: I suggested having `Homebrew.args` as a global instance kinda like `Homebrew.failed` currently is.